src/test/java/org/opensearch/security/InitializationIntegrationTests.java

```diff
@@ -46,7 +46,6 @@
 import org.opensearch.common.transport.TransportAddress;
 import org.junit.Assert;
 import org.junit.Test;
-
```

Comment:
Please remove this change if not needed.

Reply:
done